pvectorcmodule.c

```diff
@@ -365,6 +365,16 @@ static PyObject* PVector_richcompare(PyObject *v, PyObject *w, int op) {
     vlen = vt->count;
     wlen = wt->count;
 
+    if (vlen != wlen) {
```

Comment:
The same principle would also be nice to have implemented in the pure Python version of the vector.